packages/mobile/src/send/saga.ts

```diff
@@ -391,16 +389,7 @@ export function* sendPaymentOrInviteSaga({
   try {
     yield call(getConnectedUnlockedAccount)
     if (recipient.address) {
-      yield call(
-        sendPayment,
-        recipient.address,
-        amount,
-        amountInLocalCurrency,
-        usdAmount,
-        tokenAddress,
-        comment,
-        feeInfo
-      )
+      yield call(sendPayment, recipient.address, amount, usdAmount, tokenAddress, comment, feeInfo)
```

Comment:
`amountInLocalCurrency` exists to be passed into the `sendPayment` saga but it wasn't being used there so i removed it and also removed from the action as well